### DISCUSSION TITLE
1.4.0 Feature & Maintenance Release

### DISCUSSION THREAD

**Discussion by timnolte** [mattbanks/woocommerce-local-pickup-time#166] [Q&A]
### Changed

- Updated WordPress & WooCommerce Supported Versions.

### Fixed

- Updated Plugin Development Dependencies

### Added

- Added New Ready for Pickup Order Status & Customer Email
- Added Pickup Time Required & Local Pickup Link Capabilities

<hr /><em>This discussion was created from the release <a href='https://github.com/WC-Local-Pickup/woocommerce-local-pickup-time/releases/tag/1.4.0'>1.4.0 Feature & Maintenance Release</a>.</em>

**Comment by lolenkom** [comment]
Hi just to inform you that since the last update to version 1.4, this plugin cause a fatal error on all my website pages if I try to go on them to do changings. I have to deactivate this extension and the website reworks. Can you pls check the pb and update, because it was very usefull. Thx

**Reply by timnolte** [reply]
Can you open up a GitHub issue and provide more details about your installation? On what screen do you get a fatal error? What version of WordPress/WooCommerce/PHP are you using? What other WooCommerce add-ons are you using?

**Reply by lolenkom** [reply]
Sorry I'm not very useful with github. its the first time.
My wordpress : 5.9
Woocommerce 6.5.1
Woocommerce add-ons:
- Woocommerce pdf invoice & packing slip
- Woocommerce product add ons premium
- Woocommerce product bundles premium
- Woocommerce stripe gateway
- Woocommerce wholesales price premium
- WOOF Product filter
-  And yours, Local time pickup 1.4
my website : lagueulede.fr
Since I update to the last version, I got fatal error when, on "pages" list page, I click on whatever a page to edit it. I can't go further on the editing mode. White screen with fatal error from wordpress and I directly get an email alert from wordpress.
Since the last update I never add any problem with this plug in. It was just to inform you and hope there will be a solution.
Thx for your help

**Reply by timnolte** [reply]
@lolenkom have you tried deactivating any of the other WooCommerce add-ons you are using. I just checked a site that I help manage that has the latest version of the plugin along with WordPress 5.9.3/WooCommerce 6.4.1 and there are no problems. I also just spun up a quick WordPress 5.9.3/WooCommerce 6.5.1 sandbox with the 1.4.0 version of this plugin and it also isn't causing a problem. Have you added any custom hooks or calls for my plugin to attempt to change how it works?
